### PR TITLE
Merge `next` into `master`

### PR DESCRIPTION
**Pull request by jsumners**
As titled.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,33 +42,31 @@
...
-    "@types/jest": "^29.0.0",
-    "@types/node": "^20.1.0",
-    "@types/tap": "^15.0.5",
-    "esbuild": "^0.19.2",
-    "esbuild-register": "^3.4.1",
-    "fastify": "^4.0.0-rc.2",
-    "fastify-plugin": "^4.0.0",
-    "jest": "^28.1.3",
+    "@fastify/pre-commit": "^2.1.0",
+    "@fastify/url-data": "^5.4.0",
```
Comment (jsumners) [inline package.json:47]: Blocked by https://github.com/fastify/fastify-url-data/pull/147

**Comment by jsumners** [comment]
No idea what is wrong with these tests, and I have no time or inclination to figure out Jest.

**Comment by jean-michelet** [comment]
As far as I remember, it didn't always fail, but we should fix this.
Have you tried to increase the default accepted timeout @jsumners?

FYI, I'm very busy with an onboarding for the next 2 weeks and have tons of things todo/learn.
I should have more time after that, but I don't know Jest either so if you have any idea @climba03003

**Comment by climba03003** [comment]
It is the problem I faced for local environment.
I don't knows why it success in `Github Actions` before, and pop-up again now.

**Comment by jean-michelet** [comment]
Could this have been introduced by recent additions?

https://github.com/fastify/fastify-autoload/commits/next/

**Comment by climba03003** [comment]
> Could this have been introduced by recent additions?

It never run successfully in my local environment.

**Review by gurgunday — APPROVED** [review]
lgtm

**Pull request merged**